sveltejs/src/RemoteTable.svelte

```diff
@@ -28,20 +28,19 @@
     /**
      * Load data from Drupal.org API.
      */
-    async function load(_page) {
+    async function load(_page, text) {
```

Comment:
I don't think you need to actually pass the argument into the method given that the variable is scoped above the function level. Same reason pageSize and other vars aren't passed in.